c7n/resources/rds.py

```diff
@@ -405,6 +402,39 @@ def set_retention_window(self, resource, retention, copy_tags):
             BackupRetentionPeriod=retention,
             CopyTagsToSnapshot=copy_tags)
 
+    def _db_instance_eligible_for_backup(self, resource):
```

Comment:
its not clear why this was made a method, its being used across a few resource types (db clusters), so keeping it as a function is a bit nicer.